src/components/aeBanner/aeBanner.scss

```diff
@@ -0,0 +1,46 @@
+@import "../variables";
+$tablet-width: 768px;
+$desktop-width: 1024px;
+
+@mixin tablet {
+  @media (min-width: #{$tablet-width}) and (max-width: #{$desktop-width - 1px}) {
+    @content;
+  }
+}
+
+@mixin desktop {
+  @media (min-width: #{$desktop-width}) {
+    @content;
+  }
+}
+
+.ae-banner {
+  width: 355px;
+  background-image: linear-gradient(to bottom, $white, #f1f4f7);
+  border-radius: 10px;
+  position: fixed;
+  top: 40px;
+  left: 50%;
+  right: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 19000;
+
+  @include tablet {
+    width: 748px;
+  }
```

Comment:
As I see from Zeplin AeBanner don't have special behavior in tablet layout, it just shouldn't be wider than 880px in all layouts.